pages/platform/kubernetes-k8s/installing-kubernetes-dashboard/guide.en-gb.md

```diff
@@ -120,7 +126,7 @@ Using the `cluster-admin` role for your cluster, we will create a `RoleBinding`,
 To do this, please copy the following YAML into a `dashboard-cluster-role-binding.yml` file:
 
 ```yaml
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Nice catch, indeed, this resource can be updated since the version `1.8` of Kubernetes and will be deprecated for the release `1.22`. For more information, see: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#rbac-resources-v122

Reply:
Thanks